src/slidepane/SlidePane.ts

```diff
@@ -203,6 +203,8 @@ export default class SlidePane extends SlidePaneBase<SlidePaneProperties> {
 			align = Align.left,
 			open = false
 		} = this.properties;
+		// https://github.com/Microsoft/TypeScript/issues/13042
+		// const alignCss: {[Key in Align]: string} = css;
```

Comment:
Even if there is an issue, an uncomment line of code is ugly... if there is further action here when the issue is resolved, we should have an issue created in widgets, referencing this issue, and detailing what we need to do so we don't forget about it.

Reply:
Also again, seems wholly unrelated to this PR.

Reply:
Agreed. I removed this comment altogether from SlidePane. I also created #365 to track the updating of component code once TS 2.7 is released and fixes the respective issue.